modules/packages/freebsd_ports

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh -e
+
+command=$1
+
+while read -r line; do
+  export INPUT_$line
+done
+
+list_updates() {
+  pkg version -v -l "<" | sed -e 's/\([^ ]*\)-[^-]* .* \(.*\))/Name=\1\
+Version=\2\
+Architecture=none/'
+}
+
+case $command in
+  supports-api-version)
+    echo 1
+  ;;
+  get-package-data)
+    name="${INPUT_File?File must be given to get-package-data}"
```

Comment:
This will output to STDERR and exit 1 :) http://www.grymoire.com/Unix/Sh.html#uh-31 for reference